forge/routes/api/assistant.js

```diff
@@ -31,6 +50,82 @@ module.exports = async function (app) {
         }
     })
 
+    /**
+     * Endpoint to serve static assets
+     * This is used to serve static assets required by the assistant plugin
+     * namely, the models and vocabulary files (typically < 1MB in total).
+     * Assets are cached in memory to reduce load during bursts.
+     * The assets are fetched from the assistant service and cached for 30 minutes.
+     */
+    app.get('/assets/*', {
+        schema: {
+            hide: true, // dont show in swagger
+            response: {
+                200: {
+                    type: 'string'
+                },
```

Comment:
Given the assets could be binary files, not sure we should declare a `string` type here; not sure what fastify will do. If we aren't including in the api docs, then there's not much benefit to having the response object here.